SCE-Development/dev
# Finish SCE Setup Tool

## How to test this PR:
1. Run `python3 sce.py setup` in your terminal
1. The tool should auto clone SCE-RPC, Core-v4, and SCE-discord-bot and install all dependencies
1. The tool should also create an alias for the `sce` tool, so running `sce setup` from now on should do the equivalent.
1. If a directory is already cloned the tool wont clone it again.
